### DISCUSSION TITLE
Slots in data cards to be bottom aligned

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1789] [Proposals]
### Goal

To standarize the behavior of slots, when used in data cards and those cards in a carrousels, the slots should be bottom aligned
### Context

For O2ES we are using data cards  with a slot to display consumpion data, but some times the card titles are too big and create jumps as you can find in the Figma attached below

### Impact

I think that setting all slots to be bottom fixed or at least creat a prop to decide whete to place it will solve this problem

### Discussion Questions

Does this change impact any live proyects negatively?

### Do you have a visual proposal?

https://www.figma.com/design/eE87dEhhkHw8esqhfvoZ0c/Figma-para-poner-ejemplos-de-cosas?node-id=379-4190

**Comment by aweell** [comment]
Hi @MontseTelefonica !

We see the value in what you propose.

The solution of aligning the slot unilaterally to the bottom could have implications for existing production solutions, so I would avoid making that change.

On the other hand, we like the idea of making the position of the slot, or the slot + content, configurable. For example, it could be placed at the top (as it is now), at the bottom (as in the display data), or with space between (to achieve what you're showing us).

For a long time, we have been considering refactoring the whole card ecosystem to better fit the different concepts and configurations, as well as to add new ones. Before we commit to a new configuration, we would like to evaluate how this idea will impact the future of the cards.

How urgent is this petition? Can you align this with PM to prioritise this new cards study adding that configuration?

**Comment by MontseTelefonica** [comment]
Hello Alex! Our main concern are with the O2 rebrands. We used to use snap cards for this module, but now we use data cards and here is an example of something that happens when the titles don't have the same amount of characters.

<img width="346" alt="image" src="https://github.com/Telefonica/mistica-design/assets/95479574/03e17b11-4a53-4019-a904-5608f2f694b8">

**Comment by yceballost** [comment]
Se cubrirá en el proyecto Cards Evolution que trabajaremos en Q2 #2106
